### PR TITLE
feat(interaction): Add GetOption method on Application Interaction Command

### PR DESCRIPTION
**Pull request by NotYusta**
Add a utility get option function, it will help.

### CODE REVIEW

File: interactions.go [hunk 1]
```diff
@@ -326,6 +326,18 @@ type ApplicationCommandInteractionData struct {
 	TargetID string `json:"target_id"`
 }
 
+// Get the interaction data option by name
+func (d ApplicationCommandInteractionData) GetOption(name string) (option *ApplicationCommandInteractionDataOption) {
```
Comment (TheAdamSmith) [inline interactions.go:330]: Could we have these return an exists boolean as well?

**Review by TheAdamSmith — COMMENTED** [review]
Not an approver for this repo. But would love to see this get implemented!

File: interactions.go (hunk 1 above)
Comment (NotYusta) [reply interactions.go:330]: Well you can just null-check it, an exists boolean isn't needed i suppose.

**Review by FedorLap2006 — APPROVED** [review]

**Review by FedorLap2006 — CHANGES_REQUESTED** [review]
Please note the check.

**Comment by FedorLap2006** [comment]
Thanks for the contribution!

**Pull request merged**